### ISSUE TITLE
How about a manpage?

### ISSUE THREAD

**Issue by KaMiila** [nomius/ktsuss#3]
Just wondering for something that is suppose to be kept simple.... why is there zero documentation?

**Comment by AntumDeluge** [comment]
I agree. Good idea to add some documentation. Shouldn't be too much work for ktsuss. Love it by the way. I much prefer it to other su/sudo front-ends that are dependent on specific desktop environments or heavy libraries.
